tests/flytekit/unit/common_tests/tasks/test_raw_container_task.py

```diff
@@ -0,0 +1,26 @@
+from flytekit.common.tasks.raw_container import SdkRawContainerTask
+from flytekit.sdk.types import Types
+
+
+def test_raw_container_task_definition():
```

Comment:
can you do a test in your test harness, and maybe add one here, where you have a workflow with two of these raw container tasks and the second one consumes the output of the first one?  Or is that not possible to do?

Reply:
it is possible, good point. I actually have a real workflow, just scroll up. But i will write an test workflow too

Reply:
actually i cannot write a test today, as it will execute it locally, which we do not want